.travis.yml

```diff
@@ -14,9 +14,10 @@ install:
   - yarn lint
   - yarn add $TYPESCRIPT
 env:
+  - TYPESCRIPT=typescript@3.6.2
```

Comment:
suggest changing to 3.6.3 instead

Reply:
yup will do.  Somehow I managed to forget to add 3.6.x to the build matrix when I upgraded the comparison test pack.  Whoops.  Fixing now.  As far as I can assess there's no problems that have been introduced in the meantime, though I'm regenerating some of the project references test pack output again (which makes sense given the changes that have happened)

Reply:
Will push update in 15 minutes or so

Reply:
Done - now let's see how CI gets on